File: LPM.Database/Models/Relative.cs
using LPM.Database.Models.Base;

namespace LPM.Database.Models
{
    /// <summary>
    /// Родственник сотрудника.
    /// </summary>
    public class Relative : Human
    {
        /// <summary>
        /// Вид родства
        /// </summary>
        public string RelativeKind { get; set; }

        /// <summary>
        /// Дошкольник
        /// </summary>
        public bool IsPreSchoolkid { get; set; }

        /// <summary>
        /// Школьник
        /// </summary>
        public bool IsSchoolkid { get; set; }

        /// <summary>
        /// Инвалид
        /// </summary>
        public bool IsCripple { get; set; }

        /// <summary>
        /// Родственники, являющиеся сотрудниками
        /// </summary>
        public ICollection<Employee> Employees { get; set; }
    }
}

File: LPM.WebApp/src/app/modules/personnel-management/stuff/relative/relative.ts
import { BaseEntity } from 'src/app/modules/shared/models/baseEntity';
import { SelectItem } from 'src/app/modules/shared/models/selectItem';
import { Guid } from 'guid-typescript';

export class Relative extends BaseEntity {
	/* ФИО */
	name: string;

	/* Пол */
	sex: SelectItem;

	/* Дата рождения */
	birthDate: Date;

	/* Вид родства */
	relativeKind: string;

	/* Дошкольник */
	isPreSchoolkid: boolean;

	/* Школьник */
	isSchoolkid: boolean;

	/* Инвалидность */
	isCripple: boolean;

	/* Сотрудники родственники */
	employeeId: Guid;
}